src/core/chains.ts

```
@@ -1,281 +1,33 @@
-  2522: fraxtalTestnet,
-  919: modeTestnet,
-  59902: metisSepolia,
-  2358: kromaSepolia,
-  999999999: zoraSepolia,
-  44787: celoAlfajores,
-  5: goerli,
-  17000: holesky,
-  545: flowTestnet,
+  1329: sei,
+  1328: seiTestnet,
+  713_715: seiDevnet
```

Biome (1.9.4)

[error] Number literal with underscore is not allowed here.

Safe fix: Replace 713_715 with 713715


(lint/complexity/useSimpleNumberKeys)